lib/infra/infra-stack.ts

```diff
@@ -217,7 +217,7 @@ export class InfraStack extends Stack {
         initOptions: {
           ignoreFailures: false,
         },
-        requireImdsv2: true,
+        requireImdsv2: false
```

Comment:
Can we make this a top level flag? I'd like upstream this change and that's how i see this would be consumed upstream

Reply:
I'm happy to make a Jira to start a convo w/ the EE team on backporting this change, but I don't want to block our ability to make an ES 6.8 inted test on that discussion.

Reply:
Actually - I think this just falls under the existing Jira for the bug as a whole (see: https://opensearch.atlassian.net/browse/MIGRATIONS-2049).  I think it makes more sense to discuss w/ the EE team (and rest of the org) before attempting to backport.

Reply:
Makes sense, i'm good with this change for now